app/assets/stylesheets/media.scss

```diff
@@ -14,6 +14,48 @@
   }
 }
 
+.#{$namespace}-media-wrapper {
+  display: inline-block;
+  position: relative;
+  text-align: center;
+}
+
+.#{$namespace}-media-access-restricted-container {
+  left: 0;
+  position: absolute;
+  top: 15%;
+  width: 100%;
+}
+
+.#{$namespace}-media-access-restricted {
+  @include rounded($border-radius-base);
+  background-color: #ffffff;
```

Comment:
i couldn't find anything in sul_styles for white.  should i declare it somewhere in here, or ignore hound's complaint?